lib/server/index.js

```diff
@@ -24,19 +23,8 @@ module.exports = function (options) {
 		app.use(apiPath, proxy(options.proxy));
 	}
 
-	var system = {
-		config: path.join(options.path, 'package.json') + '!npm',
-		liveReload: options.liveReload,
-		liveReloadAttempts: 3,
-		liveReloadHost: os.hostname()
-	};
-
-	if(options.main) {
-		system.main = options.main;
-	}
-
 	app.use(express.static(path.join(options.path)))
-		.use(middleware(system));
+		.use(middleware({config: path.join(options.path, 'package.json') + '!npm', }));
```

Comment:
I mean being able to pass an entire `system` object into the server module. Some people could be using it programmatically.